recipes/_config.rb

```diff
@@ -0,0 +1,7 @@
+# if tls is enabled, setup dev settings by default.
+
+if node['hops']['tls']['prod'].eql?("true")
```

Comment:
This should be false. If it's not a prod environment it should set the Dev versions of the classes.

Reply:
Actually, if true then you should set the non dev classes

Reply:
Yes, you are right. Mistake. Fixed.